test/__snapshots__/katex-spec.js.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`A comment parser should not include comments in the output 1`] = `
```

Comment:
I should include the input string in the title so that it's easy to see what input produce this parse tree.